augur/io/metadata.py

```diff
@@ -40,7 +40,9 @@ def read_metadata(metadata_file, delimiters=DEFAULT_DELIMITERS, id_columns=DEFAU
         Only one id column will be inferred.
     chunk_size : int
         Size of chunks to stream from disk with an iterator instead of loading the entire input file into memory.
-
+    dtype : dict
```

Comment:
Could this say `dict or str` instead of `dict`? I only mention it since most of the use of `dtype` throughout this PR pass a single string value.

Reply:
Fixed in [force-push](https://github.com/nextstrain/augur/compare/98e82ba0ae70ff0de2374195c577de090cd1468e..4795a0bbf14bce8477daf8c0b9cd666faa383b26).